### ISSUE TITLE
P2 Composite Res: Display Aggregations in File Browser with Type Specific Icons

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#2573]
Modify the file browser interface to display folders containing an Aggregation with an icon that is specific to the Aggregation type. Can also likely do this for single file Aggregations where the file would be shown with a type-specific icon.

Additionally - the file browser interface should be modified so the Aggregation metadata file and the Aggregation OAI-ORE resource map document are not shown to the user.

Double clicking an Aggregation's folder icon will show the contents of the folder.
Double clicking on individual physical files within an Aggregation will download the file.

**Comment by aphelionz** [comment]
@pkdash Can you please add a 1,2,3 point estimate label to these tickets?

**Comment by horsburgh** [comment]
@sblack-usu - this is done in the phase 2 implementation, but I have a note that I think we should standardize icon sizes. Right now the regular folder and file icons are much smaller than aggregation icons and it looks a little unbalanced. I think we should standardize the icon sizes. Is that possible?

**Comment by sblack-usu** [comment]
I'll see what I can do.  I've got them center aligned in the auto-aggregate branch, so that should help how they look.  I'll look into resizing them as well.

**Comment by horsburgh** [comment]
OK - I should have clarified that I was thinking about the list view:

![image](https://user-images.githubusercontent.com/3485902/40125515-5aead53c-58e8-11e8-88ef-50f5d17ad51d.png)

In this example, the folders are small, the geographic feature and time series aggregation icons are large, and the referenced time series aggregation (which is a single file) is small.

**Comment by sblack-usu** [comment]
thanks for the clarification.  I had not considered the list view.  I'll see what I can do.

**Comment by sblack-usu** [comment]
I've got the list view looking good now.  I also played around with adjusting the grid view so the text aligns with the folder image.  If you don't like the grid view as imaged, I can put it back. 
![list](https://user-images.githubusercontent.com/35820390/40127424-92af1678-58ec-11e8-8659-f6690657345b.PNG)
![grid](https://user-images.githubusercontent.com/35820390/40127425-92c57e4a-58ec-11e8-893c-42c2b82be0b7.PNG)
